.github/workflows/build-kernel-a14.yml

```
@@ -94,6 +97,9 @@ jobs:
           - version: "6.1"
             sub_level: 145
             os_patch_level: 2025-08
```

```suggestion
            os_patch_level: 2025-09
```
Changing only security patch level is enough, adding new section isn't necessary.